syntax = "proto3";



message ItemPBComponent
{
	uint64 item_id = 1;
	uint32 config_id = 2;
	uint32 size = 3;
}